vis/Code/PixelTimeRange.js

```diff
@@ -55,10 +55,13 @@ class PixelTimeRange
 
 	SetAsUniform(gl, program)
 	{
-		glSetUniform(gl, program, "inTimeRange.pxSpan", this.Span_px);
+	    //Uniforms that aren't used by the shaders are commented out.
```

Comment:
Can you remove `usSpan`, `pxSpan` and `usEnd` completely from the shader instead of leaving them commented out?

Reply:
Absolutely. I left them in because I wasn't sure if this was WIP code or not.